easycheck/easycheck.py

```diff
@@ -179,6 +182,60 @@ class as a message for custom exceptions.
     check_if(not condition, handle_with=handle_with, message=message)
 
 
+def check_if_in_limits(
+    x, 
+    lower_limit = float('-inf'), 
+    upper_limit = float('inf'), 
+    handle_with=LimitError, 
+    message=None,
+    include_equal=True   
+):
+    """Check if number is in range of limits
+    
+    Args:
+        x (float): number which will be check if fit to the limits
+        lower_limit (float): the number from which we expect x to be greater
+        upper_limit (float): the number from which we expect x to be smaller
+        handle_with (type): the type of exception or warning to be raised
+        message (str): a text to use as the exception/warning message. 
+            Defaults to None, which means using no message for built-in
+            exceptions/warnings, and the docstrings of the exception/warning
+            class as a message for custom exceptions.
+        include_equal (bool): definie if limits should be included in range
```

Comment:
Let's try something like this:
"True for strict checks (lower ≤ x ≤ upper), False otherwise (lower < x < upper)"